docs/release-notes.rst

```diff
@@ -6,6 +6,62 @@
  Release Notes
 ###############
 
+**************
+ Version 0.31
+**************
+
+Version 0.31.0
+==============
+
+**Release Date:** April 15, 2024
+
+**New Features**
+
+-  API: Add a new API endpoint, ``/health``, that provides information about the status of
+   Determined's connections to the database, Kubernetes API server, and Slurm launcher integration.
+
+   See the REST API documentation for more information about the endpoint.
+
```

Comment:
Visit the :ref:`rest-api` documentation for more information about this endpoint.

Reply:
":ref:`rest-api`
documentation"

Reply:
Did this actually get added to the REST API docs page?

Reply:
`:ref:rest-api` goes to http://latest-main.determined.ai:8080/docs/reference/rest-api.html#rest-api-reference